components/cuda/install-cuda10-0.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# cuda 10.0 base    - https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/10.0/base/Dockerfile
+# cuda 10.0 runtime - https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/10.0/runtime/Dockerfile
+# cudnn7            - https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/10.0/runtime/cudnn7/Dockerfile
```

Comment:
I see that you combined all these docker files together and put here instructions. It will be difficult for us to support them because Nvidia can changes original files in the future and it could break our code.

@azhavoro , I'm thinking about a separate container which can execute some registered functions (e.g. TF annotation) using https://docs.python.org/3/library/xmlrpc.html. Could you please recommend something here?

Reply:
I agree. I was following the previous example https://github.com/opencv/cvat/blob/develop/components/cuda/install.sh
I looked like to me the `base` `runtime` and `cudnn` was combined into one file. I was unsure if this was desired on your part, maybe having less docker layers was desired.  
If there is interest in this I can write it this way. It may help with composing different cuda versions.